pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -468,6 +467,7 @@ func (s *Server) createIstioCA(opts *caOptions) (*ca.IstioCA, error) {
 
 		s.initCACertsWatcher()
 	}
+	caOpts.OnRootCertUpdate = s.updatePluggedinRootCertAndGenKeyCert
```

Comment:
`OnRootCertUpdate` is only used when the cert is self-signed. For clarity, could we move this to the above conditional block for self-signed ca certs?

Reply:
make sense